### PR TITLE
chore(deps-dev): bump @commitlint/config-conventional to 11.0.0

### PR DESCRIPTION
**Pull request by alexforsyth**
*Issue #, if available:*
https://github.com/aws/aws-sdk-js-v3/issues/1670

*Description of changes:*
Continuation of changes https://github.com/aws/aws-sdk-js-v3/pull/1672

I must bump to `node v10.22.1` in order to bump `@commitlint/config-conventional` in order to clear:
```
{
  "type": "auditAdvisory",
  "data": {
    "advisory": {
      "findings": [
        {
          "version": "3.0.0",
          "paths": [
            "@commitlint/config-conventional>conventional-changelog-conventionalcommits>compare-func>dot-prop"
          ]
        }
      ],

      "title": "Prototype Pollution",
      "module_name": "dot-prop",
      "cves": [
        "CVE-2020-8116"
      ],
      "vulnerable_versions": "<4.2.1 || >=5.0.0 <5.1.1",
      "patched_versions": ">=4.2.1 <5.0.0 || >=5.1.1",
      "overview": "Versions of `dot-prop` before 4.2.1 or 5.1.1 are vulnerable to prototype pollution. The function `set` does not restrict the modification of an Object's prototype, which may allow an attacker to add or modify an existing property that will exist on all objects.\n\n",
      "recommendation": "Upgrade to version 4.2.1, 5.1.1 or later.",
      "references": "- [GitHub advisory](https://github.com/advisories/GHSA-ff7x-qrg7-qggm)\n- [CVE](https://nvd.nist.gov/vuln/detail/CVE-2020-8116)",
      "severity": "high",
      "url": "https://npmjs.com/advisories/1213"
    }
  }
}
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by alexforsyth** [comment]
Blocked on: https://github.com/aws/aws-codebuild-docker-images/issues/399

**Comment by alexforsyth** [comment]
PR: https://github.com/aws/aws-codebuild-docker-images/pull/400

**Comment by alexforsyth** [comment]
Aws docker team is in the middle of a release. It will likely be weeks before PR: aws/aws-codebuild-docker-images#400 is reflected on the AWS cloud. In the mean time, @subinataws gave great advice about using "n 10" to pull the lastest version of node 10. Issue addressed: https://github.com/aws/aws-codebuild-docker-images/issues/361#issuecomment-717656901

Made updates in commit: https://github.com/aws/aws-sdk-js-v3/pull/1673/commits/07f9551ca05fde414d784d886287a53e806abcb2

File: buildspec.yml [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 phases:
-  install:
-    runtime-versions:
-      nodejs: 10
   build:
     commands:
       # Fake credentials for protocol tests to get past signing.
       - export AWS_ACCESS_KEY_ID=foo
       - export AWS_SECRET_ACCESS_KEY=bar
+      - n 10
```
Comment (trivikr) [inline buildspec.yml:9]:
```suggestion
      # Switch to runtime-versions nodejs 10 once codebuild upgrades its docker images
      # Refs: https://github.com/aws/aws-codebuild-docker-images/pull/400
      - n 10
```

**Review by trivikr — APPROVED** [review]

**Comment by alexforsyth** [comment]
Merging as codebuild has passed. Travis CI is failing intermittently. Unrelated

**Pull request merged**